code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -927,6 +934,15 @@
 	icon_state = "nuc_neutron"
 
 
+
+/datum/sprite_accessory/hair/fluff

```

Comment:
Add the attribute `fluff = 1` here so you don't need to re-declare it for every fluff style